src/components/ImportFromS3Dialog/get-images-from-aws.js

```diff
@@ -37,24 +38,10 @@ async function setUrl(result, configImport) {
       RecognizeFileExtension(result) === configImport.typeOfFileToLoad &&
       configImport.typeOfFileToLoad === "Texte"
```

Comment:
just a minor note: we should change instances of "Texte" to "Text" sometime in the future

Reply:
I'm sry its my french who's coming back at me